metau-capstone/StudyViewController.m

```diff
@@ -10,14 +10,79 @@
 #import "SceneDelegate.h"
 #import "LoginViewController.h"
 @interface StudyViewController ()
-
+@property (nonatomic, strong) CALayer *front;
+@property (nonatomic, strong) CALayer *back;
+@property (nonatomic, strong) CABasicAnimation *rotateAnim;
+@property (nonatomic) CATransform3D horizontalFlip;
+@property (nonatomic) BOOL isFlipped;
 @end
 
 @implementation StudyViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    //BACK SIDE
+    self.back = [[CALayer alloc] init];
+    self.back.frame = CGRectMake(0, 0, 300, 180);
+    self.back.backgroundColor = [[UIColor blackColor] CGColor];
+    self.back.position = CGPointMake(self.view.center.x, self.view.center.y - 50);
+    
+    // add text label to the flashcard
+    CATextLayer *backLabel = [[CATextLayer alloc] init];
+    [backLabel setFont:@"Helvetica-Bold"];
+    [backLabel setFontSize:20];
+    [backLabel setString:@"Back"];
+    [backLabel setAlignmentMode:kCAAlignmentCenter];
+    [backLabel setForegroundColor:[[UIColor whiteColor] CGColor]];
+    [backLabel setFrame:CGRectMake(0, 0, 300, 180)];
```

Comment:
It seems a lot of common properties set both for front and back, considering create a helper function. DRY